src/gateways/internal-api.ts

```diff
@@ -88,7 +88,9 @@ export class InternalApiGateway {
 
       return ResponseMapper.mapEvidenceRequest(data);
     } catch (err) {
-      console.error(err);
```

Comment:
I think it would be useful to keep this logging so we have visibility in the Next.js logs